spec/index.bs

```diff
@@ -3128,6 +3128,38 @@ dictionary GPUCommandEncoderDescriptor : GPUObjectDescriptorBase {
 };
 </script>
 
+## Pass Encoding ## {#command-encoder-pass-encoding}
+
+### <dfn method for=GPUCommandEncoder>beginRenderPass(descriptor)</dfn> ### {#GPUCommandEncoder-beginRenderPass}
+
+<div algorithm="GPUCommandEncoder.beginRenderPass">
+    <strong>|this|:</strong> of type {{GPUCommandEncoder}}.
+
+    **Arguments:**
+        - {{GPURenderPassDescriptor}} |descriptor|
+
+    **Returns:** {{GPURenderPassEncoder}}
+
+    Begins encoding a render pass described by |descriptor|.
```

Comment:
There is some wording missing, should say that "if attachment Xxx is provided, it's considered to be used as `OUTPUT_ATTACHMENT` for the scope of the whole render pass.

See this fragment for inspiration (but it's not ideal either!):
> If layoutBinding.type is "readonly-storage-texture" or "writeonly-storage-texture", view’s texture’s usage must include STORAGE. Each texture subresource seen by view is added to [[usedTextures]] with STORAGE flag.

Or you can just leave an Issue to describe that